kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -26,20 +26,32 @@
             {{form.stripe_public_key}}
         {% endif %}
 
-        {% if recurring_payment %}
-            <input type="hidden" id="stripe_source_setup" value="1">
-        {% endif %}
 
         <p class="legal-copy">
-            {% trans %}
+            {% trans terms_url=url('payment_terms') %}
                 Payments are not tax deductible. Your payment details will be processed by
                 <a href="https://stripe.com/" rel="noopener" target="_blank">Stripe</a>.
-                Mozilla will keep record of your payment in
-                accordance with its
-                <a href="https://www.mozilla.org/en-US/privacy/websites" rel="noopener" target="_blank">Privacy Notice</a>,
-                and does not receive your payment details.
+                You can read our Payment Terms
+                <a href="{{ terms_url }}" target="_blank">Payment terms</a>,
+                for more information.
             {% endtrans %}
         </p>
+
+        {% if recurring_payment %}
+            <input type="hidden" id="stripe_source_setup" value="1">
+            <label class="recurring-payment-confirmation">
+                {{form.accept_checkbox}}
+                <div class="control-indicator"></div>
+                <span class="legal-copy">
+                    {% trans amount='16' %}
```

Comment:
According to @atopal, we shouldn't translate legal copy, even with the "force English" hack.